FluidNC/src/Kinematics/PolarArm.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2020 -	Bart Dring
+// Use of this source code is governed by a GPLv3 license that can be found in the LICENSE file.
+
+#pragma once
+
+/*
+	WallPlotter.h
```

Comment:
Should be PolarArm.h

Editorial comment: This is an example of why I dislike including filenames in comments.  They are frequently wrong due to failure to update them.